Makefile

```diff
@@ -37,7 +37,9 @@ ifndef HAS_GOLANGCI
 endif
 	golangci-lint run
 
-doccheck:
+doccheck: generate
+	@echo "- Checking if the generated documentation is up to date..."
+	@git diff --exit-code --
 	@echo "- Checking if the documentation is up to date..."
```

Comment:
Can we change this to something like `Checking if the documentation is in sync with the code"` ? Do you agree that this would perhaps add more clarity on what's going on ?

Reply:
Makes sense, done ✔️

Reply:
Whoops, I actually meant changing this `"- Checking if the documentation is up to date..." `to `"- Checking if the documentation is up to date..."`

The informational line that you had added in this PR was fine IMO.

Reply:
ha, fixed and unfixed. Should be good now.